dockerfiles/Dockerfile-mina-rosetta

```diff
@@ -46,6 +46,7 @@ RUN apt-get update --quiet --yes \
     && apt-get install --quiet --yes --no-install-recommends \
     apt-utils \
     apt-transport-https \
+    bc \
```

Comment:
Utility needed for load script. This docker already contains all testing tooling so that's why i decided to add it here. in the future we would like to introduce rosetta-slim docker which will be free from all those auxiliary tooling